config/locales/en.yml

```diff
@@ -35,7 +35,7 @@ en:
   feedback: "Feedback"
   issue_contact: "Please contact the SUCCESS Center at (843) 792-8300 or success@musc.edu for assistance with this process or with any questions you may have."
   portal_title: "SPARC Request Portal"
```

Comment:
Sorry, you're right. I had the button turned off locally and didn't see the problem. Looks good!